en-US/Punctuation.xml

```diff
@@ -426,19 +426,43 @@
 	 <section id="quotation-marks">
 		<title>Quotation Marks</title>
 
+<!-- Previous guidance
 		 <para>
 			Commas and periods go inside quotation marks.
 		</para>
  		 <para>
-		Question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside. 
-		</para>
-	</section>
+			Question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside. 
+		</para> -->
 
-<!-- Not implementing this change for now.  		
 		 <para>
-			In technical documentation, punctuation marks, including periods, commas, question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside.
+			In technical documentation, punctuation marks, including periods, commas, question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation or part of a programming element that uses quotation marks; if not, they go outside the quotation marks. If a sentence ends with a quotation, use only one punctuation mark to end the sentence, and place it inside the closing quotation mark if it is part of the quotation, or outside if it is not.
+
 		</para> 
-Add examples of when to use or not to use quotation marks. 
+		<example><title>Correct Examples of the Use of Punctuation with Quotation Marks</title>
+        <para>
+        For bind mounts, add "bind" and set fstype to "none".
+        </para>
+		<screen>
+		Login successful.
+		...output omitted...
+		Using project "default".
+        </screen>
+		<para>
+		As you can see in the previous image, the title of the web page is "OpenShift 3 Etherpad".
+		</para>
+		<para>
+		In the program segment, ensure that the value is X'FF'.
+        </para>		
+		<para>
+		If you use the keyword operand PGMID="PAYCOM", you must ...
+        </para>
+		<para>
+		Julius Caesar said, “I came. I saw. I conquered.”
+        </para>
+      	</example> 
+
+	</section>
```

Comment:
It's a good idea to stay on top of formatting, indentation, etc., when working in markup languages. It can make a difference to readability, troubleshooting, and in some cases (e.g., YAML) actual results.

Reply:
In my VS Code the indentation looks the same regardless of whether a leading tab or spaces are used. That appears not to be the case in this view, so I'm changing those lines to use a tab only.